AmplifyPlugins/DataStore/Podfile.lock

```diff
@@ -122,4 +122,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 04860e414d616b67d24ed3346a60700f427764b9
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.10.0
```

Comment:
nit: does this change belong to this PR?

Reply:
Actually, I am curious what impact does this change have?